### ISSUE TITLE
Is it possible to get older WhatsApp messages?

### ISSUE THREAD

**Issue by alexolmait** [bhsoftdata/All-in-one-WhatsApp-scraper-and-sender#1]
Is it possible to get older WhatsApp messages? - that appear after pressing the  button “Click here to get older messages from your phone.”

**Comment by bhsoftdata** [comment]
Yes possible to do this as custom charge apply. please contact us at
***@***.***

**Comment by alexolmait** [comment]
Let's estimate the custom charge.
The provided email address can not be seen :( . can we communicate directly
by mail? or other means, whatsapp, viber 995599585211?

**Comment by bhsoftdata** [comment]
Ok email us at bhansalisoft781986 at gmail.com

**Comment by bhsoftdata** [comment]
On product detail , our Conatct email given .. please contact me on email
directly.
